device-registry/src/test/scala/org/genivi/sota/device_registry/DeviceResourceSpec.scala

```diff
@@ -47,15 +47,23 @@ class DeviceResourceSpec extends ResourcePropSpec {
     case None => false
   }
 
-  property("GET, PUT, DELETE, and POST '/ping' and /system_info request fails on non-existent device") {
-    forAll { (id: Id, device: DeviceT, json: Json) =>
+  property("GET, PUT, DELETE, and POST '/ping', /system_info and /group_onfo request fails on non-existent device") {
```

Comment:
I think these should have their own `property` and not be crammed inside a single property